bin/splunkversioncontrol_utility.py

```diff
@@ -36,7 +36,7 @@ def runOSProcess(command, logger, timeout=60, shell=False):
     return str(stdout), str(stderr), res
 
 # use the password endpoint to obtain the clear_password passed in, start with the context of this app and then try all contexts
-def get_password(password, session_key, logger):
+def get_password(password, session_key, logger, *, sslVerify=False):
```

Comment:
Overall this looks great but why the ", *," ? Is this similar to *args? But why is it there?
I can see it in a couple of places

Reply:
It can be safely removed. The single asterisk argument is a python 3 feature which forces the following arguments to be named.

This means you cannot call this function just with `False` as an argument you must explicitly specify `sslVerify=False`.

My bad if you still want compatibilty with python 2 it must be removed as the python 2 parser will throw an error.

Reply:
That's a trick I had not seen. I still want python 2 compatibility, if you have time to update the pull request I can merge.

Or I can update it tomorrow

Reply:
I can do it this evening. (Central Europe Time)